packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts

```diff
@@ -10,8 +10,15 @@ export async function pushResources(
   context: $TSContext,
   category?: string,
   resourceName?: string,
-  filteredResources?: { category: string, resourceName: string }[],
+  filteredResources?: { category: string; resourceName: string }[],
 ) {
+  if (context.parameters.options['iterative-rollback']) {
+    // validate --iterative-rollback with --force
+    if (context.parameters.options.force){
+      throw new Error("'--iterative-rollback' and '--force' cannot be used together. Consider runnning 'amplify push --force' to iteratively rollback and redeploy.");
```

Comment:
Can you set the error stack to `undefined` here - it doesn't seem like the stack trace will actually be useful.

Reply:
Added both errors related to iterative rollback in it's own error class.

Reply:
Does that mean this should be an `IterativeRollbackError` now and not just an `Error`?